app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/task.scala.html

```diff
@@ -59,6 +59,6 @@ <h2 class="app-task-list__section">
         </span>
         @getTag(section.status)
     </li>
-    @section.footnote
+        <li>@section.footnote</li>
```

Comment:
should this not be inside of the first li? this will create two per section

Reply:
yes, agree. done